node-notes-cli/app.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
+const dataParsed = require('./data.json');
+
+function read() {
+  for (const property in dataParsed.notes) {
+    console.log(`${property}: ${dataParsed.notes[property]}`);
+  }
+}
+
+function create() {
+  dataParsed.notes[dataParsed.nextId] = process.argv[3];
+  dataParsed.nextId++;
+  fs.writeFile('./data.json', JSON.stringify(dataParsed, null, 2), err => {
+    if (err) throw err;
+  });
+}
+
+function remove() {
+  for (const property in dataParsed.notes) {
+    if (process.argv[3] === property) {
+      delete dataParsed.notes[property];
+    }
+  }
```

Comment:
You can skip the loop here since you have `process.argv[3]` on hand.

```js
const property = process.argv[3];
delete dataParsed.notes[property];
```